elastic/kibana
# [Security Solutions] Add telemetry to the side navigation tracking for serverless pages

issue: https://github.com/elastic/kibana/issues/164306

## Summary
Add <TrackApplicationView> telemetry to serverless-specific Security pages 
Pages: Investigations, Assets, ML, Project Settings.

## How to test it
* Visit Investigations, Assets, ML, and Project Settings.
When you navigate between pages the app should log "Reporting application usage for securitySolutionUI, {PAGE}" like it does for the other pages
